kobo/apps/accounts/tests/test_email_content.py

```diff
@@ -0,0 +1,131 @@
+        )
+        email_subject = EmailContent.objects.create(
+            email_name='email_confirmation_signup_message',
+            section_name='subject',
+            content='This is a test subject line'
+        )
+
+        # Using the randomint to make sure that usernames and emails
+        # are unique to ensure the tests will pass when run back to back
+        # as only one email can be requested per username and email every three
+        # minutes
+        username = 'user001'
+        email = username + '@example.com'
+        data = {
+            'email': email,
+            'password1': username,
+            'password2': username,
+            'username': username,
+        }
+        request = self.client.post(self.signup_url, data)
+        user = get_user_model().objects.get(email=email)
+        assert request.status_code == status.HTTP_302_FOUND
+        self.client.login(username=user.username, password=user.password)
+        self.client.get(reverse('account_email_verification_sent'))
+        assert len(mail.outbox) == 1
+        assert mail.outbox[0].subject == email_subject.content
+        assert email_content.content in mail.outbox[0].body
+
+    @override_settings(CACHES={'default': {'BACKEND': 'django.core.cache.backends.dummy.DummyCache'}})
+    def test_custom_activation_email_template_blank_content(self):
+        email_content = EmailContent.objects.create(
+            email_name='email_confirmation_signup_message',
+            section_name='section_one',
+            content=''
+        )
+        email_content_closing = EmailContent.objects.create(
+            email_name='email_confirmation_signup_message',
+            section_name='section_two',
+            content=''
+        )
+        email_subject = EmailContent.objects.create(
+            email_name='email_confirmation_signup_message',
+            section_name='subject',
+            content=''
+        )
+        # Using the randomint to make sure that usernames and emails
+        # are unique to ensure the tests will pass when run back to back
+        # as only one email can be requested per username and email every three
+        # minutes
+        username = 'user002'
+        email = username + '@example.com'
+        data = {
+            'email': email,
+            'password1': username,
+            'password2': username,
+            'username': username,
+        }
+        default = "Thanks for signing up with KoboToolbox!"
+        request = self.client.post(self.signup_url, data)
+        user = get_user_model().objects.get(email=email)
+        assert request.status_code == status.HTTP_302_FOUND
+        self.client.login(username=user.username, password=user.password)
+        self.client.get(reverse('account_email_verification_sent'))
+        assert len(mail.outbox) == 1
+        assert mail.outbox[0].subject == 'Activate your KoboToolbox Account'
+        self.assertNumQueries(1)
```

Comment:
This doesn't do anything. It needs to be used as a context manager, see django docs. Generally when testing, please ensure the test works as intended by making it fail. In this case, no number will ever fail.